Networking/Networking/Model/Product/ProductAttributeTerm.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+/// Represents a `ProductAttributeTerm` entity.
+///
+public struct ProductAttributeTerm: Equatable {
+    public let siteID: Int64
+    public let termID: Int64
+    public let name: String
+    public let slug: String
+    public let count: Int
+
+    /// Member wise initializer.
+    ///
+    public init(siteID: Int64, termID: Int64, name: String, slug: String, count: Int) {
+        self.siteID = siteID
+        self.termID = termID
+        self.name = name
+        self.slug = slug
+        self.count = count
+    }
```

Comment:
nit: I think this initializer isn't needed? (there is a default one that is the same as this one)